binaries/daemon/src/pending.rs

```diff
@@ -77,7 +77,7 @@ impl PendingNodes {
         clock: &HLC,
     ) -> eyre::Result<()> {
         if self.local_nodes.remove(node_id) {
-            tracing::warn!("node `{node_id}` exited before initializing dora connection");
+            tracing::info!("node `{node_id}` exited before initializing dora connection");
```

Comment:
Maybe we should handle dataflow-specific warnings in a different way in the coordinator? For example, instead of printing them to stdout, we could forward them to the coordinator, which then forwards it to the CLI (if requested). We could for example integrate it into `dora logs` this way (e.g. show dataflow-level warnings if no node ID is specified).